README.md

```diff
@@ -71,5 +71,5 @@ after checking the current joint states (to create minimal increment for safety)
 joint. The commands are incremented in regards to /joint_states found when the node is run.
 
 ```
-ros2 run ur_ros2_control_demos test_driver 
+ros2 run ur_ros2_control_demos test_driver
```

Comment:
I have one small request before merging:

Please add the basic instructions to get CI passing to the README here. Like, what clang format command to run?

Suggest to do it in this style:

```
## Contributors

Code should be formatted with clang-format 10 before submitting like this:  `clang_command`

Add any other tips here, too
```

Reply:
I pushed a small commit to add this